### ISSUE TITLE
encoding error when reading source on windows

### ISSUE THREAD

**Issue by trim21** [afonasev/flake8-plugin-utils#9]
* Date you used flake8-plugin-utils: 1.0.0
* flake8-plugin-utils version used, if any: 1.0.0
* Python version, if any: 3.6.8
* Operating System: windows

### Description

like https://github.com/terrencepreilly/darglint/issues/46 .

the default encoding of python file is utf8, and `open` will try to read a file and decode it as `gbk` on windows.

### What I Did

create a utf8 encoded python source file contents CJK like `测试用例`

use `Plugin._load_file` to read it.

got
```
UnicodeDecodeError: 'gbk' codec can't decode byte 0x97 in position 1345: illegal multibyte sequence
```

Soluton: 

read it as bytes, and pass bytes to `ast.parse`.

```python
    def _load_file(self) -> None:
        with open(self._filename,'rb') as f:
            content = f.read()
            self._lines = content.splitlines()
        self._tree = ast.parse(content)
```

or I write a package: https://github.com/Trim21/read_source

**Comment by afonasev** [comment]
Thx for issue! I will try to fix this soon.

**Comment by afonasev** [comment]
Fixed in 1.1.1 version
